### DISCUSSION TITLE
No arrow back on lenses

### DISCUSSION THREAD

**Discussion by StanBarrows** [laravel/nova-issues#4705] [Q&A] [answered]
- Laravel Version: 9.*
- Nova Version:  4.12.*
- PHP Version: 8.1.*
- Browser type and version: Chrome

### Description:

In the documentation there is a back arrow within a Lense, but on my application there is no arrow visible.

https://nova.laravel.com/docs/4.0/lenses/defining-lenses.html

![CleanShot 2022-08-17 at 16 05 43@2x](https://user-images.githubusercontent.com/10268813/185154786-379337cf-ccb5-4160-903a-e1157725f66d.png)

![CleanShot 2022-08-17 at 16 07 32@2x](https://user-images.githubusercontent.com/10268813/185155228-db32cc62-e753-43c3-9014-24dbdaa40607.png)

### Detailed steps to reproduce the issue on a fresh Nova installation:

- [ ] Create a Lense with some metrics above

**Comment by StanBarrows** [comment]
Why was this moved into the Ideas & Feature Requests?  It's within the documentation but not within the app?

**Comment by davidhemphill** [comment, marked as answer]
The screenshot shows an older version of Nova. This was removed in version 4.

**Comment by StanBarrows** [comment]
Thanks for the update.The Screenshot is still present in the 4.0 Documentation.

**Reply by davidhemphill** [reply]
Just fixed the screenshot. Thanks for bringing it to our attention!

**Comment by StanBarrows** [comment]
@davidhemphill  would be cool to have this error back.
